rebasehelper/checkers/abipkgdiff_tool.py

```diff
@@ -144,17 +149,25 @@ def parse_changes(lines):
             (?P<what>Added|Changed|Removed)(\s+functions|variables)?
             (\s+\((?P<filtered_out>\d+)\s+filtered\s+out\))?
             ''', re.VERBOSE)
-            result = {}
+            result_list = []
+            title = 'Undetected soname'
```

Comment:
But this isn't [`soname`](https://en.wikipedia.org/wiki/Soname), it's filename of a shared object.